core/theme.js

```diff
@@ -57,27 +57,44 @@ Blockly.Theme = function(name, blockStyles, categoryStyles,
    * @private
    */
   this.componentStyles_ = opt_componentStyles || Object.create(null);
+
+  /**
+   * The font style.
+   * @type {?Blockly.Theme.FontStyle}
+   * @package
+   */
+  this.fontStyle = null;
 };
 
 /**
  * A block style.
  * @typedef {{
-  *            colourPrimary:string,
-  *            colourSecondary:string,
-  *            colourTertiary:string,
-  *            hat:string
-  *          }}
-  */
+ *            colourPrimary:string,
+ *            colourSecondary:string,
+ *            colourTertiary:string,
+ *            hat:string
+ *          }}
+ */
 Blockly.Theme.BlockStyle;
 
 /**
  * A category style.
  * @typedef {{
-  *            colour:string
-  *          }}
-  */
+ *            colour:string
+ *          }}
+ */
 Blockly.Theme.CategoryStyle;
 
+/**
+ * A category style.
+ * @typedef {{
+ *            family:string,
+ *            weight:string,
+ *            size:number
```

Comment:
From the way these are used I think it should be `string?` and `number?`.

Reply:
Ideally it would be string|undefined but closure doesn't support that in typedefs :(

Reply:
I can make it nullable though.